package.json

```diff
@@ -6,29 +6,41 @@
   "module": "dist/styled-media-query.es.js",
   "jsnext:main": "dist/styled-media-query.es.js",
   "repository": "git@github.com:morajabi/styled-media-query.git",
-  "keywords": ["styled-components", "media-query", "media", "breakpoint"],
+  "keywords": [
+    "styled-components",
+    "media-query",
+    "media",
+    "breakpoint"
+  ],
   "scripts": {
+    "test": "jest",
     "clean": "rimraf dist",
     "build": "yarn build:common && yarn build:es && yarn build:browser",
     "build:common": "rollup -c --environment COMMON,PRODUCTION",
     "build:es": "rollup -c --environment ES",
     "build:browser": "rollup -c --environment BROWSER,PRODUCTION"
   },
-  "files": ["dist", "src"],
+  "files": [
+    "dist",
+    "src"
+  ],
   "author": "Mohammad Rajabifard <mo.rajbi@gmail.com>",
   "license": "MIT",
   "peerDependencies": {
-    "styled-components": ">= 1 < 4"
+    "styled-components": ">= 4",
```

Comment:
Do not see any reason to remove support for styled-components < 4.

So, let's have `">= 1"` here.

Reply:
I agree here.

Reply:
OK, I concede ^^.